samples/nrf9160/aws_fota/Kconfig

```diff
@@ -10,38 +10,50 @@ config APP_VERSION
 	string "Application version"
 	default "v1.0.0"
 
+config USE_NRF_CLOUD
+	bool "Use nRF Cloud certificates"
+	default n
+
+if !USE_NRF_CLOUD
 config CLOUD_CERT_SEC_TAG
-	int "Secure tag fo TLS credentials"
+	int "Secure tag for TLS credentials"
```

Comment:
In LTE Link Monitor, this is called security tag, not secure tag. Which is correct? We need to be consistent in terminology.

Reply:
Seems we should call it security tag that's what it's called in the zephyr documentation my bad.

Reply:
Should we change it here or in the PR for updating the doc?

Reply:
I'll add it here.